src/compas_robots/model/base.py

```diff
@@ -132,6 +132,9 @@ def scale(self, factor):
         """
         self.point = self.point * factor
 
+    def to_compas_frame(self):
```

Comment:
plus this gives you a copy of the wrapped frame rather than the frame (if that could be an issue)

the wrapped object is available via `self._proxied_object` maybe it makes sense to create a couple of accessors in the parent `ProxyObject`.
suggestions:
- `unproxify()` - no copy
- `unproxfied()` - yes copy

Reply:
Yes, I will remove this function all together. I vaguely remember something didn't work and I made this function, and probably after finding out this is not the problem, I didn't remove it.